rlp-derive/tests/rlp.rs

```diff
@@ -10,62 +10,62 @@ use rlp::{decode, encode};
 use rlp_derive::{RlpDecodable, RlpDecodableWrapper, RlpEncodable, RlpEncodableWrapper};
 
 #[derive(Debug, PartialEq, RlpEncodable, RlpDecodable)]
-struct Foo {
+struct Item {
```

Comment:
lol, I didn't know it this lint existed https://rust-lang.github.io/rust-clippy/current/index.html#blacklisted_name
Not sure how useful it is though